azurerm/resource_arm_servicebus_namespace_authorization_rule_test.go

```diff
@@ -0,0 +1,190 @@
+package azurerm
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/servicebus/mgmt/2017-04-01/servicebus"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	"strconv"
+)
+
+func TestAccAzureRMServiceBusNamespaceAuthorizationRule_rights(t *testing.T) {
+	cases := []struct {
+		Name   string
+		Rights []string
+	}{
+		{
+			Name:   "listen",
+			Rights: []string{string(servicebus.Listen)},
+		},
+		{
+			Name:   "send",
+			Rights: []string{string(servicebus.Send)},
+		},
+		{
+			Name:   "send-listen",
+			Rights: []string{string(servicebus.Listen), string(servicebus.Send)},
+		},
+		{
+			Name:   "manage",
+			Rights: []string{string(servicebus.Listen), string(servicebus.Send), string(servicebus.Manage)},
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.Name, func(t *testing.T) {
+			testAccAzureRMServiceBusNamespaceAuthorizationRule(t, tc.Rights)
```

Comment:
whilst this makes it more readable - this means it's not possible to run these tests individually; can we split them out as regular tests?